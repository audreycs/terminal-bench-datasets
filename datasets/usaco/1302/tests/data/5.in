100

9 312312779 388648827
243136582 595807543 273753407662876805
914401543 454675286 371469710668626906
531662581 391504819 314211220759898896
5259587 93487486 37285475048334951
238137372 792053951 354468340149171532
467468186 355772948 265054584241634591
840331508 878197388 505988792609996613
883068287 209997650 317655161689343255
828797118 149643530 256087622178231942

5 910439502 567147616
82627194 846627320 407629659816695499
383594814 206331238 407344153654226474
837861796 113576498 807645232571421929
787351220 817745415 1177804853039092451
927735236 399571499 1043232019401534435

6 719711015 525107491
705986560 88019797 490173099997341800
185339872 197453447 148168991505944037
575173083 477241592 443490593804359542
472437647 496381821 376928341345243910
761191595 122343221 552896556903911175
400825261 374106923 332478601016528168

8 872913212 737640606
730189533 949834826 719245678192388880
888229207 786188898 1235283189841305835
853137252 81454818 733200494233122446
286556030 644920931 620689585815327193
467089149 979565489 955890172670714602
592240840 270786627 454091102058233058
118231129 337305257 274537620686648420
909748423 473994743 1012128642963486921

7 439127656 800239605
335385761 408151430 397987151098581843
45857253 287865163 203424940958122078
620280405 239037378 392592172150351411
749999154 986465364 717120587627964908
346862270 572715294 524174268083769840
136778065 430776270 296992997267319328
415350060 439548768 440755994134766621

7 502182088 207651477
976942941 891650612 626870307664661734
476322321 520596192 316602931542566370
990828193 572667746 593443052762207370
677962975 727272576 446205010862548018
306448121 283145485 200495165011518333
942607523 88546553 482256097424733321
3674374 79716666 17512283483717980

9 650116790 341351500
68879078 960712436 368749716911016108
356379263 237299962 259993453189426686
88825557 822617967 254037462430142097
901969061 41430990 451891241477544435
381905327 856914212 397870084983246739
573572480 673687450 451807109456248797
368296587 325362253 287575717829966563
387624923 587837769 378459237284245521
636860989 856131629 651408929406088623

9 761155532 151986982
727898393 40858304 485355576252050318
874807971 322044883 685560836206549822
165375025 230580330 94326675238827296
632163749 224010342 421914889976182153
907051545 472295443 736452877852781827
40915855 689604354 80333320637239652
663775920 765804411 524855610118434159
291702354 646494277 254994447670207135
467358183 680118343 443873643939544273

6 26477768 936756582
597536317 377446153 88602578385476259
99308134 246466237 71853625414881747
337598349 394243657 340584399061491360
150847780 651030450 451000178444356365
785123189 845706315 419609678378888127
320754903 730713572 207860785066748346

8 719224251 232941558
823172788 854655506 686839903204946095
842352772 926389446 631574686420786044
291203995 460352309 243516584533455945
35049782 261185803 79089853146847471
789326589 590842687 689278044602086724
348334338 273069711 303937789184883291
978758366 983768141 892769076619789197
335642504 832092217 381502576218137460

6 970495941 204503578
199498075 579807107 227764755765405383
353253744 326419348 330617468672655082
340331975 990325669 487841187051493796
544846257 570264214 503637038906502786
685643117 963288621 798190047122539254
97423509 360976519 148931888073860386

8 941459797 160622302
807585207 403169854 823007077251468153
937098494 475905682 956008414843530379
889622300 106570346 854024715303421436
236628560 877146497 362649025149062553
330879516 229692140 345926865798005882
802417246 321848033 806248175771636996
800568807 999937046 909186061266193381
532913133 666449259 606381182426506025

8 531035616 402365628
389968889 719981189 324693861745830025
654659592 222137938 273658706863796628
567962939 160169629 246392162779977259
973466849 486104207 554117621274885135
112608540 497431936 131903357982406989
3070130 899390086 316126704436187488
192460610 139687054 121184729518569754
166940468 305041830 134663314125663284

6 366065293 911619683
97002032 69153357 87940733377012462
716053330 903718730 799272346895059390
810318210 286599568 466749749653767472
407177209 759538116 495663770253011775
789354975 962063341 968342943681118999
164519166 449268864 406148977278518322

5 138642815 847655335
40133621 640172174 507648399426953260
211962104 258470891 234942367036689177
22972962 71487617 61704660641744456
276443118 501012357 444930351542296642
811116932 451087931 468605763416720760

7 350049616 488584483
695094210 258932826 358787987706866562
445281944 124866691 214236341926606789
908897043 722658384 586699934149314610
947281886 504013853 525577807366593402
192696634 340757706 201698391776146645
967832536 610774148 536639206319742201
561989170 522160584 387002559511322636

6 184603129 462798415
647910387 28834981 77552428403319038
542896122 196537137 117534226401880044
762669632 807240389 310252840249497881
129794181 814590931 342752429842750003
195997736 835129645 144023471944388830
2537656 155296773 18757314630228869

8 956338858 807503121
234429838 128116087 300877129391741199
628951966 332327834 452792984631350091
788756315 57755782 496094615297935486
693350088 330527831 781664845111738474
925393414 75678110 202897719325710539
399474076 628841082 848312004499205141
696808692 183324865 709060050781964326
985572161 308589518 485034914782513797

7 752148330 599381551
251725699 254814200 165446952824798695
33038461 500861326 148380261194354190
815209563 263201262 410132267109805311
540223301 631695772 695072436949835106
422978275 971399476 355529740039256406
147576268 572350254 449698084949541918
79740861 395500469 187236352760155770

7 524816575 320395631
435270510 445170645 302968958086159912
616302901 998334806 369770418801464675
3796166 710968881 221487626258521515
674013320 351573653 325406627742825983
21987063 609213031 189847405639164548
238918503 31034520 130500159999423745
957187081 533169587 578527591423228723

5 234151889 929857460
531306133 305537378 398194156940371431
707500494 286403251 380135961175929342
539822504 313550152 371145777889780447
144794285 340450926 339564072866461365
892045104 961746620 1090543265810720536

6 580164503 905765312
296132723 778272529 808914533668155081
452141380 766478187 941264438423806350
572486431 389670333 472259104193636988
506725589 910271355 995395454743471041
520976523 87240746 284595180380628191
537891980 21711623 288688208941882522

7 202342039 930381367
976224017 247122226 410547821073642245
385395116 894659104 821611273608829912
133234114 98499533 108192405783179743
365047451 653324500 659597330930477136
763544607 952327938 948370937712272644
138728504 212679549 217494937900631651
17273215 565668135 521036175009627564

6 356198476 296474991
649759482 364181013 136790116575868495
236586827 365438831 122590336259079862
125942 172435448 33755586333460562
607382327 751274514 390638820557529837
939486031 327987570 340622671759409546
107257477 975743664 262389748587514037

6 253587897 329337223
803529029 232074193 245229301916187438
28794799 221378986 50983866761826144
31143143 659371712 107047121320410741
102217498 221431883 98697770094919222
679880284 3208045 162784742168085411
475699027 238731349 158066366525652685

6 384161515 798416820
99158314 407678201 332631662133605688
34294406 211183607 176316118083507543
839416514 602965018 778547805710433341
654336944 740773027 769088057245014750
787156896 74568814 358464356224392697
992513642 264650998 544317359222126293

8 205552842 808874368
707918255 244139343 290119657416428313
98670077 429756799 329814436710834353
345722492 455840333 302602235641938087
952978761 993844029 389267255331308307
961689864 9671796 127472961001124548
828940865 381070459 297720793789716318
918542975 389194683 296477606235660422
705284792 496005131 205940090493117514

7 293722576 948480412
176905738 266536084 146556969247039202
246867428 874769933 846142269369295165
840187434 482573936 538895996779932113
985463138 263079668 219520165492212573
489087321 610186874 325625617307219791
172723827 441079844 311040254423269414
143202329 457355186 285443697276361314

9 220540347 368189740
729450838 400388810 141727930374501153
717417827 948557856 161272963998130221
461525343 858999152 416310992548696248
759010259 625916778 147366986450894553
857444310 360709826 206747362571165850
992305793 335790295 185600281648563762
810810789 293869972 131774290720344045
949414532 922037459 537113715481307531
899209592 898232444 482507417825912050

7 981796340 508384995
196365328 33602902 169511424316639610
796696865 392159800 842410501708047474
886985612 742283818 1159418950058132653
135114647 930876701 596688976185014590
925731033 550942978 1080926540287792766
910341308 576142399 975495881458812518
54709939 621179428 335420150710625400

7 504795721 208005780
999404511 176061754 409351901300547630
526023510 280800809 294783410602333539
82146887 291151522 62772692982706188
351400959 301204658 145297979673919933
927755786 123598029 338120924639920091
411208991 959347066 359143773340291392
700339806 346008841 250476725135774890

6 443494727 348524450
709619233 277326504 173236021322189438
921202479 495409351 471006270407826491
402968331 523787200 187149723883391842
281278162 924136218 235394615666162121
79520210 331350221 84988234347443043
960997915 623266757 250421860921705397

9 438866175 181529749
366995181 471576449 171716668780380171
33227154 508222151 67553829433181991
489613528 736778384 301896207624829306
553444786 341707103 278676675764365957
650551095 517410488 373519909056695931
150824906 5704912 63550908774858122
268536611 292170041 160680042103120552
301138234 281677329 168177615050789078
379480832 600544896 165217474015482515

9 629174590 806883407
155842750 221162574 249102039069005983
397862873 643802333 672402414892605741
26892176 825247267 638252125615712894
423010056 204501668 387181002228453547
304907437 234545354 361721296948053619
67912776 7284945 45636044332662843
132314666 43460196 107273736214498490
115474759 51996817 114463180956060541
238635451 896835101 795724280730781662

10 901974288 147506263
506953961 367930477 446825204716433985
137931725 553579417 193596952970997658
469687340 894949736 413755759482984358
791807279 922572187 723414950312801108
970439793 615284584 886300146702746588
324429877 483147401 317994565929085834
193556846 379945764 204760044920533292
455646750 401224524 415630523524690897
720956750 487397830 704088179340554033
170468310 754543454 229530635257126298

6 904556925 116666724
648333380 551311593 628763663195433674
169870032 767010865 218751275054550018
93370904 409236689 97517560396935656
317489199 422991793 280253532121164167
176407516 108011844 149963349023527431
150622934 790198551 216748431307647457

5 452614963 37247199
557288030 853116797 259322726530525902
13340039 656400723 26112001378688619
786573463 715856329 330865715825573335
879221315 83455067 385730894066694281
72523173 394138183 47401128314357016

10 444475212 998627959
899055160 713197832 684130164969614340
913077665 397261278 51339061832386684
587893572 58059255 31702074031829196
912271123 688323418 494977401588206634
881388202 691444375 542837229185776332
656495544 377672497 498546840193564622
327225866 937814104 895687109671755161
742278356 854655323 196693447351377364
767733606 672303280 372676472885954711
796042813 966748518 351991979432826367

6 751406495 667978316
281130939 785187381 539633812929565566
17827332 801557395 501727467064903695
245274502 873782065 675410452497197246
891641221 937884151 915934740182358268
629152117 456546540 754887201200067297
107384310 273506602 206267702084079196

7 834104290 961739459
938341745 757638758 1097740576099357958
577486409 934128337 1283204030597450178
221640737 157649466 167791957064703671
856593072 80606101 631425209531894172
943324142 717297582 918586223703413380
357336901 801615722 750375292012650182
143552699 16601010 127045904112070398

9 802712274 680223542
559942882 863019031 745633295426523549
358164389 967094140 421323586749342044
493310137 237213928 502524429239632857
324764267 466748886 494674393444256550
536809007 392530716 672860331201100164
61937324 49861461 53602675414100264
38483094 786218050 434884201735216158
520551753 317978397 580397249407145251
481856585 685169549 675989356316928708

10 60677148 718649795
302090130 724597742 347802761879601250
631343095 192530459 135409511646168165
302907842 151947519 63022973256425580
10364779 751216289 202405016452174504
790449709 765820447 248158521882941492
186272097 207580233 137229084715342909
133156463 515906907 341746857202132966
129287192 162103817 118331298108806911
471036367 902810510 245344321108786750
25787205 744266643 217221380581210284

7 120225809 729017977
681327183 125401646 167913865696451250
276154303 788211893 415251858530810754
116185575 788203064 398922397364087682
909830669 922919499 639390058307040220
174038380 621553144 391446106992389463
749257066 90637456 139541109569742443
74624946 173811442 114524712237810807

6 334019992 302112590
519174923 448928482 258667704602915270
583049273 41100674 155448985284735093
564701423 116320489 121224455014233266
717261240 159776177 240462907086215507
912445484 614895886 311500254372987680
499734128 944021914 395499308133786472

8 398289048 573949976
757217830 481503379 551158069519088692
511508557 330692961 208851637709467746
36110071 68589210 35226321036738189
597556257 63033714 222256279625846499
186861046 749020994 397340766803622936
536758274 781104285 645252303532252969
782470788 897442118 750238714621648617
278199881 621637929 336668707005624021

8 634139670 193893178
729692632 204153172 399867051849510593
463135366 261028284 304084584788076372
881068654 396937251 555417476601852333
334716029 164453015 136233289322117096
69669910 696435891 139942896975265075
559522376 237545619 147380868899086742
519199240 617045097 360998841816647002
142793216 126287058 48735214718236720

5 956154929 564636067
354687909 635402231 506157037114604093
106118701 105012052 153573937393515178
688211161 622304777 372974727254053333
337491816 951264431 841132700042960179
573159188 611325644 622240773697371374

5 754886798 342802533
903662513 299003417 637010870165462061
335664444 982539738 523385973803723375
985938663 674001052 486591413974683086
372081809 331716173 341648801067798770
78770867 577714347 236192445840878642

7 25516602 505831118
885508848 890324850 338790955628792865
287047779 325946502 118892315244203616
463690513 800632763 297279418398722117
892936209 151584959 93039022721683749
920268950 983783447 406785481130063775
136069905 223261448 94552579553040107
220241035 978297802 488739260076286708

5 701599624 595029761
953350461 785183002 982359983782125144
784735180 802529806 458801122139519156
154902206 152351758 84957022143942207
924032458 59393710 333497741795297207
918170908 280431039 259585938921589244

9 138839975 908072901
257512005 409811376 399513786245147026
388981862 271702203 266376972901397225
516636469 62131383 125066846830904367
464584561 890250558 750365270696004051
476788389 234602735 257443472380652562
942458886 997561648 871836569942775811
895139592 625818952 655546223498825513
50332373 928295779 715165952870829845
104528730 725793908 552502121087165822

9 145792846 347997501
70540016 76547369 34895426331191941
318939123 572900245 234832619143933513
504325501 342813562 177003293537987665
630724323 289198851 185612375823067656
313269841 469051899 189990031986974080
927742547 286523110 232164051101238710
884765879 114082059 162576783693090584
974675359 343210989 244526280229314747
11124066 17405229 7064707429101628

10 606315241 992174855
941554880 877649917 1426341300345569144
754971872 336954241 624207364775915441
158035560 540711062 566799904259743648
488968502 224433200 451568644196188367
474508781 906863346 916587947234159163
229224517 891134799 942903618579130756
915939143 535126463 1042237808372571968
60267103 636053175 578898101295914508
701324815 961784587 1215630461983416210
566721957 810714812 1075143966854873443

8 351856504 227177143
758244722 642212963 348852114155796881
411139585 604795751 250947989097962521
971072240 359258597 388799893200370738
768210372 247615296 232826552502490947
540153533 393687645 166958764587798873
719923286 251334514 257495578942552908
748843178 704885480 272737013966185107
799023858 633799118 276434161863162827

10 68274596 408112204
283585808 280608091 94475620808696273
435233122 622554465 247490432222472331
535119497 342533237 173081899470091351
198536796 621377721 132325301839663248
272292875 68757467 45703886685172202
665192075 180906696 63097325548799536
769053415 46896179 44230838783775886
437365055 279573506 94024590618476686
359177471 550104529 156940874467495554
787891798 796771486 164503042124008796

9 110480611 61760839
140699078 37454823 16191959594325249
218680157 657724695 61924997483881513
810087805 294149759 61251435173678583
639389713 899617212 115188280172416913
13260304 5383900 1613132348474037
963967880 915140064 118223180410656980
396790941 453113677 40613976235630132
250609803 775701581 45612756664462931
564462471 708371919 88838720409380617

6 712194312 303695874
803384177 506429624 566554113359157366
129694013 543269596 146070933221536217
132377550 172435312 93173745844343144
11897015 413365773 127398507250533047
619010912 426246279 216853099566284424
694462980 2917873 142304898350762826

6 640824694 626229008
553994144 999539908 667648391891714228
159079614 430630 97949423828458035
682574246 188124235 433169921647663866
855381049 462103730 585692167906933922
757397625 520258233 637750114818157320
342113599 537078227 283383650098102196

9 956316959 289166911
840830596 879730387 326630411400228824
553351106 167707928 475651882202633080
50299408 459140523 121442043881720090
371087612 471630640 474097368889663240
474742028 655871920 448681336301216857
100481682 428154842 199308310332491398
439154277 105888558 328981416658790022
587980717 650262764 274132807966100065
127697622 356933347 137552150626327156

5 139012187 146420254
826461836 491028367 127786479924357628
78528832 670119594 87697506989541698
363746898 569447239 76477236168099509
47520402 134731495 13247796146171405
537716711 636654986 159928416820415144

10 404918522 348765978
386021082 848200866 373970974295914430
661765671 856734662 533978724559795750
189865449 612311306 288995903172857392
922928806 53445212 370188695946018997
820571422 917802242 622633817481768576
158530892 14920197 48852515788176958
15489858 447801966 161568730111679945
378187097 385871959 254715406571919770
5362839 536399542 186003884141033878
541547185 659508645 337364710775763684

9 204716009 11762766
40404157 524617634 10505198036332520
705261478 628887425 102630111745512285
561141355 377992318 85300708840597070
740223307 467800765 138863280121385089
136148648 634038333 29435903828188300
954746901 220111873 183422379157385741
551920494 958911399 84459411183758379
493810977 266965279 80026757154126559
560070734 936415231 98824607839908086

9 404794503 221449563
455075331 541896192 177793137376782378
68878078 496935431 127373084674592697
344639155 820726889 177940724609277718
466371398 94974583 119484792637971972
748318872 443940420 388507711447013696
715206773 779695379 291862528041544309
606272175 522165786 201245593576077423
603574457 155470861 167556706603893810
603304520 131783809 265239246045383031

5 656169310 996089372
440739531 992801403 879973471769916560
68119602 664814908 513941985264071256
830299263 547109758 599994112441484106
581686127 868720981 500823519893648897
371880729 955716142 1054588251391059990

5 528324695 616565530
196789363 532428298 379186598556403606
870576183 821408119 759361328579808809
244041915 829797990 639955731615403367
205797829 974020866 589140735929725215
621234498 447722634 354308700357693526

9 570933566 876842698
12965472 170548924 136764917281330564
483626727 615469847 800491818007301872
562516026 675271736 801009779352964227
354947774 865921721 958788225791706378
739667254 624280095 812718894635457883
157330650 613301315 372359922898983981
607154971 896927626 899184700701036600
886314815 805084330 911373369421878741
853477538 399219563 566756844281182792

6 728434463 721578390
418890037 653922325 431941623139280655
699574366 38742053 383611042782763092
977777836 705757766 723183067680306490
264789451 731173696 559961928485974267
68906223 577679869 399764847730390144
227489517 771951980 372254065609756997

7 576179012 413590689
562529268 113057321 342451476466023736
608501910 35536476 355610137446076035
254870283 386704384 208858355390205777
741307720 282591099 509046963924600274
485788246 940590283 563807416573664877
412575149 639812316 412180197457835832
696122848 295510793 445333840290086201

7 334424841 569998743
967827086 508238539 592848951014815296
137401669 984290771 598814379561869846
116385593 751994868 464623008258685747
433621541 518355259 425527147577452228
293100605 590094973 429945494474250084
663193204 876880900 713847687186753515
46503951 10201974 20619619978263774

7 924429116 407011365
60781372 902350724 423304033706405394
187258204 67076544 107411031626970440
533823495 754849835 728989475361201200
100670776 437374715 167193837906950586
28741346 391189405 116487023342408362
244948246 399720864 338411560616808482
231468488 453412914 376395053111477210

8 604617425 303097761
782592001 630831741 473570433932615870
961604490 864351182 436106280915226143
233553176 237513001 167514878329633744
591644793 628845713 435452743603822164
702419687 313681369 350565767948211804
958108394 159839135 421844414896295837
13466088 305696070 95598201286440805
954370290 164936818 517904180885799267

5 563358037 402342540
731009574 380178068 414668304620153722
315673280 762634114 438229805592561039
335339842 775520080 482683314963091697
64918286 543923740 251127236517298066
705030344 492436109 504146346709862231

9 384938779 97719131
319085356 142516528 133849391149537487
374016131 76574545 150528750773602679
199316378 444181982 104753279576023039
615292455 426991777 205834627010587525
167765199 372740200 76864626943316793
861114339 721765189 333401455621528221
57036774 916644441 98746773975077683
627409473 539828575 210913774010851778
942787842 248480082 344393750291763287

7 155329058 385795331
854445697 440366895 197168573398493038
165459968 877796453 80405082545869456
134177185 976719596 201758777837420150
450917499 584196395 203479989983725305
220349495 991745493 167219474476755269
374764439 465476261 144649774048923375
760099354 469151272 211492100784334172

7 236351464 431723314
441145373 160799481 100010381200147186
173636330 489083772 229240773980887792
256232063 781141873 111811662731552065
678568775 570995153 195265366795442421
877410884 286137208 326071304807492816
287102996 529140953 169699048335261534
94263922 682399064 215197367274204965

9 73411578 602449408
182125680 925835725 537335424928356548
336817171 522116332 245171109723959079
357368284 251116537 138246796197191588
137947134 348508238 179898841260288875
431239356 326380056 186045835223208792
703936348 497788367 269817185883439279
749579932 60092083 77482874939705772
597520565 121499114 83272028487021123
650220576 3172827 39066023593442723

7 862055463 247947603
117088576 709013101 262760046578303515
461164405 190469406 421600774141271712
240418549 41832133 130554681985135191
537520837 153023202 418708563320539699
306253362 100986393 199174869006009003
267326661 659059488 362068014721125150
359099426 939198446 459523494836818265

5 65236880 531954675
617411996 73286370 64283661380260902
541022636 721874395 329710379923951274
700574293 596348271 320657147010717685
820654997 196310087 156616835927742320
190287201 424014267 191404438524040608

6 132071409 933925754
590097117 799079902 715388687634811385
660387156 746889589 756584563787081339
402999854 961799889 914189897451855913
688667821 108699958 190487146933615522
835651233 75273056 170136246691603199
243232886 56437638 74848776241128554

5 65225072 178055343
483373418 543813911 124302514376000227
773119363 570581611 147608704507991495
108817542 403382405 44274631625828656
955590150 236027816 83014949307519467
137667271 576782273 61849674545391999

7 799078566 452776632
819994777 638674151 286649174515754038
538305474 797845728 719515075764558374
77850696 660040084 265299769557240585
596873101 855421308 550519518400775666
938285212 856075555 206560707969322810
57718032 578761373 293903646639227250
23368355 273315457 22217986845306262

10 503007996 252557894
66272012 767363077 174288388458276941
470645738 45431600 229621738227183952
30897806 263189169 57012287921721494
914031503 966519842 645833503750552787
219065085 523906744 200280996421663931
498937732 82854751 264995042622376951
120257549 538561070 169436810481426196
668257788 195793549 311280408417121724
766347287 487041904 486774123304301582
777405687 410546957 438092709033652195

9 799685377 342668302
248673846 440723302 247018483497475292
411811976 416164054 414489143720779228
348045493 617861530 293741058650505498
741158321 847007737 770799174606450748
55929734 30083510 47242200024215152
294362129 634500288 273402550149835991
899841336 807844813 860323904765634608
856433085 872177400 711541418305429319
187400858 474824979 255327364566952790

5 364671355 142236709
98913514 668342506 81316780952880896
23572415 416989830 66520015080529225
943367779 196808891 239481780702536087
168948927 322260401 101980918776186587
835227785 380544367 250192204002320430

7 281427783 815144486
433335230 728184139 166846546541085026
159142707 216724260 44705694808448329
558692832 349708551 336816573372314888
741017417 98918876 49073608246476165
670443924 386642261 472819074997668475
561403920 646249568 124499136861416864
573745311 919156725 555519724349931669

7 825591774 558258859
744850351 287433991 753797580528267432
260095879 19344708 225145730444320609
890912341 86147487 778695761087203297
612547186 441494248 735448441684311788
630225403 597574890 841378503732022340
330771279 364270206 469823135005704598
833928454 988601120 1233002516735115428

9 59905353 44313246
819848403 224765212 52501330586130540
843431048 611321326 68019166645884480
763485980 778547936 44491756706977973
427330538 640653950 48893196540736855
42142827 899678516 36436637309545930
686139574 489081189 32428791355138022
533175101 662479776 47911768470159096
320650722 361997369 22120771554724095
155531143 5279901 8529914060703270

9 553635552 587706066
965065903 91390173 358942767205675004
186700476 828581494 453068489041839937
69885052 87644654 79536033150371775
509449339 159457927 196708374251038127
815096175 127651058 455067948135969195
856853666 543360151 486325659358662637
252012531 168833302 203155419830171014
214964738 317858928 273739117923426667
259323550 20355726 138828216800505896

9 437995399 275900867
83303420 379938454 50400784217683909
251042559 403580044 129168111280735886
467474644 150449083 223382485541744028
712440925 863403087 321965206807148095
16711708 917829255 66546887266002077
611853907 832606377 472811342546411546
173179383 129077313 107705709890496160
592702354 913825220 505403340302708134
729405487 554046943 334110496672978609

9 288237732 286989062
136159762 320339640 127262137479873701
268445219 488121306 214401904058865179
864082427 193994507 291682862866226689
900950905 295054314 338663372815013898
942301050 856001561 488076332924290423
440789709 372075865 202472628768866622
101426278 871691468 252941511162056877
889565269 128286982 287936946489036075
138025221 300533366 109142700195218780

8 518685899 119893597
814738691 583482312 419007919277982347
222846017 913890881 160239446685323487
502995214 171184656 204363827892868993
488689827 226870555 206148856323007005
763360188 297741045 297423786781104284
484943463 732721806 276989131691547641
323519230 742621776 184806654644766786
426887958 734001915 282554146984283620

10 844818198 402614687
296894104 399172272 231519807343374454
171839576 834198284 411136761320957539
669060467 657163470 452541149306992143
692205839 662745388 831263395456900946
151667057 930645909 402758682656428580
41051542 979034132 352683977196221980
682306793 429239605 438407541147704585
242275213 521978379 264645634715001289
841630111 477754037 708958952950239652
775657073 461505817 357799682348007250

8 214249252 953086813
377317047 457205974 457087405575914813
907106770 49421086 188790252185628106
173815122 379571203 362840965451426080
444657459 918631976 916713127816366965
242329651 736367702 429149607503737599
995122409 855174059 922080772766354023
208629120 350831997 358140432640497249
354420894 739765388 567669720019509041

7 748308778 624436874
880936116 618595981 1029578853696211268
289879168 156493164 312095264997687306
482956465 397706219 553441243640174672
781879023 205810107 598584706676904072
696310627 894558844 1064976730458437978
402538675 767178742 700541208588875403
954730148 895754061 1143443872783704689

7 171924904 725919758
420559046 166947768 132466792439750051
208863899 898764470 658112460169184456
939300348 580765908 392786041257995167
700570319 299813326 290817226649642353
527499037 618170208 462496814636237583
583984227 628249302 426746358653297197
513026619 658468206 246418301087245778

10 276306392 12436176
588685834 496573828 155755964977555208
241429345 136029963 62772194550449257
895634753 245316916 218345270621980164
372891148 168631857 97489273439233682
19787877 817975544 12611953417017418
171505044 676168493 47568032184102840
930484311 481949674 212573054153718927
949257469 927054951 232857327705511414
426064395 456792235 112335998185851635
30023896 108933050 8800529657007221

5 124161910 262869345
272669824 940299555 112210705668448611
108654420 757295772 135716204294954702
101675538 798758751 200712782510961969
102401976 52115006 5751204442636253
873167664 962344853 187962243979667123

6 138482533 58093887
666667942 7391599 58145460337663143
448615230 402869758 62307662024335999
797039616 512238849 132735889668819718
859013492 213270259 123966817148497476
859192277 318433109 121717622251192387
80934428 759040491 50166558476116244

5 399882689 334889025
330609047 235190570 193092687831460974
376515035 987958763 457739190322420671
503176307 9181359 196495641621174016
230319932 16445181 95036903098678536
332697227 179996415 185126576544629980

5 614867396 516548314
256982114 486994563 308013306668892095
428606078 26795976 229222134066462721
947438511 737914973 804147358294034677
97086663 450567912 253185691699124526
300011159 460259137 364145182501034456
