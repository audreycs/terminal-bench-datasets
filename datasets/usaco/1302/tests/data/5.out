114944090
231398205
453248433
703375598
407143982
62566918
255708170
342762101
728492050
205162588
250634100
7484829
393338067
455276126
63359826
124481077
414083314
803120339
732661517
311375498
86716706
371759156
103193519
356143283
193134377
102901782
620660317
688280195
384839138
245368983
287724636
454628609
190306285
168073066
168563133
153561149
66359489
1313870689
405739387
785730128
585515496
478551596
244312041
196222757
403902824
465581102
924770659
495700141
163542336
738992091
166830313
42730915
354077110
208338180
269392011
74759752
596534276
524018688
857022255
108327696
191743088
69996778
277479091
873086256
414980194
519542117
617012029
255378925
30790159
589342876
423541552
205351105
133982267
323475256
391727668
188364792
362166809
126726466
139172355
87194354
1025771811
171188434
317720694
184375599
897907698
29008934
46651899
415372790
264412190
77137819
194806782
644547539
464053246
169970657
485640505
55459525
276675240
53129349
55398104
257037966
